install_config/aggregate_logging.adoc

```diff
@@ -1942,10 +1942,10 @@ test-project:
 ----
 
 
-When you make changes to any part of the EFK stack, specifically Elasticsearch
-or Fluentd, you should first scale Elasticsearch down to zero and scale Fluentd
-so it does not match any other nodes. Then, make the changes and scale
-Elasticsearch and Fluentd back.
+To make changes to Fluentd, change the configuration and restart the Fluentd pods to apply the changes.
+To make changes to Elasticsearch, you must first scale down Fluentd and then
+scale down Elasticsearch to zero. After making your changes, scale Elasticsearch first and
+then scale Fluentd back to its original setting.
```

Comment:
A note that we generally apply softwraps as we encounter them in existing docs. (A cmd+j should do the trick if you're using Atom.) Not necessary to apply here but just a heads up.